Quoridor.Model/AlivePlayer.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Quoridor.Model
+{
+    internal class Player : IPlayer
+    {
+        public Player(Wall[] walls, Position start, Position end, Position currentPlace)
```

Comment:
Place constructor after fields.
Also You can name parametrs as fields, but with lower letter
Like (CurrentCell = currentCell, EndPosition = endPosition) and etc.